<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>964</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>预览</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="0">
                <widget class="QWidget" name="videoLabel3" native="true">
                 <property name="styleSheet">
                  <string notr="true">image:url(:/Homework/Resources/logo.png);
border:1px solid gray;</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QWidget" name="videoLabel1" native="true">
                 <property name="styleSheet">
                  <string notr="true">image:url(:/Homework/Resources/logo.png);
border:1px solid gray;</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QWidget" name="videoLabel4" native="true">
                 <property name="styleSheet">
                  <string notr="true">image:url(:/Homework/Resources/logo.png);
border:1px solid gray;</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QWidget" name="videoLabel2" native="true">
                 <property name="styleSheet">
                  <string notr="true">image:url(:/Homework/Resources/logo.png);
border:1px solid gray;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTableWidget" name="tableWidget">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>150</height>
             </size>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <column>
             <property name="text">
              <string>时间</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>报警信息</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>设备类型</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>录像回放</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QGroupBox" name="playBackGroupBox">
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>报警与布防</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="addDeviceAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="settingAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>录像回放</string>
    </property>
    <addaction name="localPlayBackAction"/>
    <addaction name="timePlayBackAction"/>
    <addaction name="namePlayBackAction"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>报警与布防</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addDeviceAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/add.png</normaloff>:/Homework/Resources/add.png</iconset>
   </property>
   <property name="text">
    <string>添加设备</string>
   </property>
   <property name="toolTip">
    <string>添加设备</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="settingAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/setting.png</normaloff>:/Homework/Resources/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="recordAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/startRecord.png</normaloff>:/Homework/Resources/startRecord.png</iconset>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
   <property name="toolTip">
    <string>开始录像</string>
   </property>
  </action>
  <action name="localPlayBackAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/play3.png</normaloff>:/Homework/Resources/play3.png</iconset>
   </property>
   <property name="text">
    <string>本地录像回放</string>
   </property>
  </action>
  <action name="timePlayBackAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/startRecord.png</normaloff>:/Homework/Resources/startRecord.png</iconset>
   </property>
   <property name="text">
    <string>按时间回放</string>
   </property>
  </action>
  <action name="namePlayBackAction">
   <property name="icon">
    <iconset resource="Homework.qrc">
     <normaloff>:/Homework/Resources/play.png</normaloff>:/Homework/Resources/play.png</iconset>
   </property>
   <property name="text">
    <string>按文件名回放</string>
   </property>
   <property name="toolTip">
    <string>远程文件回放</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>布防</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>撤防</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Homework.qrc"/>
 </resources>
 <connections/>
</ui>
